docs/HowTo/Get-Started/Register-Validators.md

```diff
@@ -0,0 +1,129 @@
+description: Register validators     
+<!--- END of page meta data -->
+
+# Register validators
+
+Register validators by creating validator keys or use existing keys to deposit funds
+to a deposit contract on an Ethereum 1.0 node.
+
+The following instructions allow you to register validators in a private network.
```

Comment:
Suggest making active: 
Use the following instructions to register validators in a private network.

Reply:
Done